CefSharp/Structs/ScreenInfo.cs

```diff
@@ -7,10 +7,40 @@ namespace CefSharp.Structs
     public struct ScreenInfo
     {
         public float ScaleFactor { get; private set; }
-        
+
         public ScreenInfo(float scaleFactor) : this()
         {
             ScaleFactor = scaleFactor;
         }
+
+        /// <summary>
+        /// Height of the monitor nearest to the window hosting the browser
+        /// </summary>
+        public int Height { get; set; }
+
+        /// <summary>
+        /// Width of the monitor nearest to the window hosting the browser
+        /// </summary>
+        public int Width { get; set; }
+
+        /// <summary>
+        /// Available Left location  of the monitor nearest to the window hosting the browser.  iF toolbar is on the left - this would be a non-zero value
+        /// </summary>
+        public int AvailableLeft { get; set; }
```

Comment:
Use Rect to represent the 4 values https://github.com/cefsharp/CefSharp/blob/master/CefSharp/Structs/Rect.cs

Maybe the AvailableRect property nullable?